gammapy/modeling/parameter.py

```diff
@@ -23,7 +23,12 @@ def _get_parameters_str(parameters):
         else:
             value_format, error_format = "{:10.3f}", "{:7.2f}"
 
-        line = "\t{:12} {:11}: " + value_format + "\t {} {:<12s}\n"
+        line = "\t{:21} {:8}: " + value_format + "\t {} {:<12s}\n"
```

Comment:
why do we use this `_get_parameters_str` ?
It would be simpler to just show `parameters.to_table().pprint(max_lines=-1, max_width=-1)`.